ui/src/router.js

```diff
@@ -243,4 +199,74 @@ router.beforeEach((to, from, next) => {
   }
 });
 
+function checkPageRole(roleName,routeName)
+{
+  var resCheck=false;
+  switch(routeName)
+  {
+    case "search-people":
+      if(searchPeopleAllowedRoles.includes(roleName))
```

Comment:
I think having the roles in a variable name is going to get unwieldy as this project grows. One day, we might even want this to be configurable as a FHIR resource. Because each variable is tied to only one page anyway, using a variable doesn't really get you anything here.

Also, what happens if checkPageRole is called but doesn't match any of these switch statements?

I think you could simplify this to something like:

```
let roles = [];

switch (routeName) {
  case 'search-people':
    roles = ["Admin", "Edit", "View"];
    break;

  case 'add-people'
    roles = ['Admin', 'Edit'];
    break;

   ...
  }

  return roles.includes(roleName) && roles.length;
```